jwst/tests_nightly/general/nirspec/test_calwebb_spec2_nrs_msa.py

```diff
@@ -32,7 +32,7 @@ def test_msa_missing(self, caplog):
         with pytest.raises(Exception):
             Step.from_cmdline(args)
 
-        assert 'Unable to open MSA FITS file (MSAMETFL)' in caplog.text
+        assert 'Mising MSA meta (MSAMETFL) file' in caplog.text
```

Comment:
"Missing"

Reply:
Though, if there's a misspelling here, then probably in the message in the actual function as well?